### DISCUSSION TITLE
not being able to run v2 blank project.[mac, Android Studio] Namespace not specified

### DISCUSSION THREAD

**Discussion by meehjeevan** [axmolengine/axmol#1354] [Q&A]
I was trying to run the blank project of v2 on mac Android Studio but I got stuck in here. It says Namespace not specified. 
but I have put namespace in build.gradle of app folder. 
Sorry, if it may not be directly related to axmol v2 but I couldn't test the blank project. Can anyone help plz?
![Screen Shot 2023-09-25 at 4 54 26 PM](https://github.com/axmolengine/axmol/assets/6400139/51401b07-4bd7-4bd4-8f33-b91ec3d1181a)

**Comment by halx99** [comment]
Update to latest dev branch will fix your problem, relatived commit: https://github.com/axmolengine/axmol/commit/3e0d1a3ae9d74daaade75e3f4c652c781dff76ee

**Comment by meehjeevan** [comment]
tried with latest dev branch too, but still I'm not able to run the project. 
![Screen Shot 2023-09-25 at 11 22 08 PM](https://github.com/axmolengine/axmol/assets/6400139/02fb39e2-7d68-42c0-9fdd-814439b4d71b)

**Reply by halx99** [reply]
did you recreate new project?

**Reply by halx99** [reply]
and: https://github.com/axmolengine/axmol/discussions/575

**Reply by meehjeevan** [reply]
Yes, I recreated new project.. it wasn't either at that time..
haven't checked with the new latest update yet..
